apps/store/extensions/app/social-reviews/modules/reviews-utils.js

```diff
@@ -43,37 +46,71 @@ var ReviewUtils = {};
         paging.limit = paging.limit ? paging.limit : REVIEW_PAGE_LIMIT;
         return paging;
     };
-    var processUserReviews = function(socialSvc, user, reviews) {
+    var processUserReviews = function (socialSvc, user, reviews, target) {
         var review;
-        var formattedUsername = formatUsername(user);
+        var myReview;
+        var socialSvc = getSocialSvc();
+        var username = formatUsername(user);
+        myReview = JSON.parse(String(socialSvc.getUserComment(username, target)));
+        if (!isEmpty(myReview)) {
```

Comment:
As we have defined getUserComment method [here][https://github.com/wso2/carbon-store/blob/master/components/social/org.wso2.carbon.social.core/src/main/java/org/wso2/carbon/social/core/ActivityBrowser.java#L41] in ActivityBrowser interface , It will return a JsonObject, In jaggery side, it will receive as stringified JSON object , so in case of no comment it will be an empty string not undefined